tests/testthat/error/test-subscript.txt

```diff
@@ -3,51 +3,60 @@ vec_as_subscript() forbids subscript types
 ==========================================
 
 > vec_as_subscript(1L, indicator = "error", location = "error")
-Error: Must subset with a proper subscript vector.
-x `i` has the wrong type `integer`.
-i The subscript must contain names.
+Error: Must subset elements with a proper subscript vector.
+x The subscript has the wrong type `integer`.
```

Comment:
I think the lack of symmetry in these messages (integer vs names, character vs locations) is likely to cause confusion, and simply using the type name in the info bullet would be adequate. If you really wanted to continue using indicators/locations/names, I'd suggest putting them in parentheses.

Reply:
You're right, and we don't like "indicator" so much. Probably best to keep it simple. I'll make the changes in another PR.